TWLight/templates/new_base.html

```diff
@@ -10,6 +10,26 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
 
+    <meta property="og:title" content="The Wikipedia Library" />
+    <meta property="og:type" content="website" />
+    <meta property="og:image" content="https://upload.wikimedia.org/wikipedia/commons/6/63/Wikipedia_library_logo.png" />
+    <meta property="og:image:width" content="200" />
+    <meta property="og:image:height" content="200" />
+    <meta property="og:url" content="https://wikipedialibrary.wmflabs.org/" />
+    <meta property="og:description" content="{% comment %}Translators: Information about The Wikipedia Library.{% endcomment %}
+    {% blocktranslate trimmed %}
+    Over 90 of the world's top subscription-only databases,
+    with content in {{ no_of_languages }} languages,
+    free for Wikipedians of all backgrounds
+    {% endblocktranslate %}"
+    />
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:image" content="https://upload.wikimedia.org/wikipedia/commons/6/63/Wikipedia_library_logo.png" />
+    <meta property="twitter:site" content="@WikiLibrary" />
+    <meta property="og:locale" content=en_US />
```

Comment:
use the value from [django.utils.translation.get_language_from_request()](https://docs.djangoproject.com/en/3.2/ref/utils/#django.utils.translation.get_language_from_request) instead of hardcoding `en_US` Django will take care of figuring out the best locale to use.

Reply:
Thanks, I have removed the hardcoded `en_US` and used the current language here